### PR TITLE
[browser] Fix tabview margins. Fixes jb#53673

### PR DESCRIPTION
**Pull request by mkenttala**

### CODE REVIEW

**Comment by mkenttala** [comment]
Got notified there's already PR that is likely to fix this one too: https://github.com/sailfishos/sailfish-browser/pull/817

File: apps/browser/qml/pages/components/TabView.qml [hunk 1]
```diff
@@ -43,7 +43,7 @@ SilicaGridView {
     onCountChanged: if (count > 0) closingAllTabs = false
     onClosingAllTabsChanged: if (closingAllTabs) closeAllPending()
 
-    width: parent.width - Theme.horizontalPageMargin
+    width: parent.width - 2 * Theme.horizontalPageMargin
```
Comment (rainemak) [inline apps/browser/qml/pages/components/TabView.qml:46]: Other way around. Both SilicaGridView and PullDownMenu should expand to full width and only the actual content should have y:  Theme.horizontalPageMargin and width: parent.width - 2 * Theme.horizontalPageMargin

Comment (rainemak) [reply apps/browser/qml/pages/components/TabView.qml:46]: That implying also that background texture used in private browser mode should expand to full width.

Comment (xlise) [reply apps/browser/qml/pages/components/TabView.qml:46]: Please don't need to fix it, I'll fix it in https://github.com/sailfishos/sailfish-browser/pull/817

**Comment by mkenttala** [comment]
Closing, will get fixed with #817.

**Pull request closed without merging**